contrib/golang/filters/http/source/cgo.cc

```diff
@@ -200,6 +200,19 @@ CAPIStatus envoyGoFilterHttpSendPanicReply(void* r, void* details) {
   });
 }
 
+CAPIStatus envoyGoFilterHttpSetStringFilterState(void* r, void* key, void* value, int state_type,
+                                                 int life_span, int stream_sharing) {
+  return envoyGoFilterHandlerWrapper(r,
+                                     [key, value, state_type, life_span, stream_sharing](
+                                         std::shared_ptr<Filter>& filter) -> CAPIStatus {
+                                       auto key_str = referGoString(key);
+                                       auto value_str = referGoString(value);
```

Comment:
nope, I think copyGostring for value could be better.

we now need twice memory copy for value, when it's in the Go thread,

continue in following.